packages/create-fuels/tsup.config.ts

```diff
@@ -0,0 +1,3 @@
+import { indexBinAndCliConfig } from '@internal/tsup';
+
+export default indexBinAndCliConfig;
```

Comment:
Add the following to `internal/tsup/src/index.ts` immediately _before_ `export const indexBinAndCliConfig`:

```ts
export const bindAndCli: Options = {
  ...tsupDefaults,
  entry: {
    bin: 'src/bin.ts',
    cli: 'src/cli.ts',
  },
};
```

Then delete the `create-fuels/src/index.ts` file and:

```suggestion
import { binAndCli } from '@internal/tsup';

export default binAndCli;
```